### PR TITLE
Landing page cleanup

### PR DESCRIPTION
**Pull request by vunderkind**

### CODE REVIEW

File: apps/web/components/Carousel/GuidesCarousel.tsx [hunk 1]
```diff
@@ -65,24 +71,46 @@ const Wrapper = ({ hits }) => {
...
+    </div>
+  );
 };
 
 const Guides = connectHits(Wrapper);
 
 const GuidesCarousel = () => {
   return (
-    <div className="relative h-[200px]">
+    <div className="cursor-pointer">
```
Comment (Necmttn) [inline apps/web/components/Carousel/GuidesCarousel.tsx:113]: can you add this to proposal cards as well

**Review by Necmttn — CHANGES_REQUESTED** [review]
cam you also compress dark mode bg images they are about 300 & 500 kb per file. compare 20-40 kb light mode ones. I believe we don't need to include the black background in the image itself, @ianbach  can you help us to export without bg?

File: apps/web/components/Carousel/GuidesCarousel.tsx (hunk 1 above)
Comment (ianbach) [reply apps/web/components/Carousel/GuidesCarousel.tsx:113]: Yes I can compress - unfortunately we do need the BG in them due to how I build the noise gradient in Figma :(

Comment (Necmttn) [reply apps/web/components/Carousel/GuidesCarousel.tsx:113]: if I'm not wrong you could have the noise bg on transparent bg then we place it on black bg via CSS then we can have the same result on the app?

Comment (ianbach) [reply apps/web/components/Carousel/GuidesCarousel.tsx:113]: Learn_d
propose_d
vote_d

Comment (ianbach) [reply apps/web/components/Carousel/GuidesCarousel.tsx:113]: > 

unfortunately not - it's to do with how I've achieved the grain effect. In future I can explore how to make this transparent but I think for now we just use jpgs

Comment (Necmttn) [reply apps/web/components/Carousel/GuidesCarousel.tsx:113]: Ah gotcha understood 🙌
thank you.

**Pull request merged**